desktop/api/src/main/java/org/datacleaner/util/IconUtils.java

```diff
@@ -188,6 +188,10 @@
     public static final String DICTIONARY_TEXTFILE_IMAGEPATH = "images/model/dictionary_textfile.png";
     public static final String DICTIONARY_DATASTORE_IMAGEPATH = "images/model/dictionary_datastore.png";
 
+    public static final String NEWS_CHANNEL = "images/news/news_channel_16.png";
+    public static final String NEWS_CHANNEL_ORANGE = "images/news/news_channel_28.png";
+    public static final String NEWS_CHANNEL_HIGHLIGHTED = "images/news/news_channel_20.png";
```

Comment:
Also a bit of a problem with the filenames as well: From the name alone, I would think that they represented exactly the same image in 3 different sizes, but that is not the case.

That reminds me, remember to put the high-res/scalable originals in the NCIM/graphics project so we can reuse them for other things or if we need other sizes.